pkg/reconciler/kubernetes/tektoninstallerset/install.go

```diff
@@ -150,6 +150,13 @@ func (i *installer) ensureResources(resources []unstructured.Unstructured) error
 			return err
 		}
 
+		if res.GetDeletionTimestamp() != nil {
+			err = fmt.Errorf("waiting for resource %s: %s/%s to be deleted",
+				r.GetKind(), r.GetNamespace(), r.GetName())
+			i.logger.Infof("found resource in deletion state, waiting! %v", err)
```

Comment:
@l-qing can you please change the log level to `Debugf`

Reply:
OK, I thought the likelihood of this happening was low, so I used the info level for logging.
It's only happened twice in three years.